src/data/finset/basic.lean

```diff
@@ -2914,6 +2916,18 @@ variable [decidable_eq α]
 theorem to_finset_card_of_nodup {l : list α} (h : l.nodup) : l.to_finset.card = l.length :=
 congr_arg card $ (@multiset.erase_dup_eq_self α _ l).2 h
 
+lemma disjoint_to_finset_iff_disjoint {l l' : list α} :
+  _root_.disjoint l.to_finset l'.to_finset ↔ l.disjoint l' :=
+begin
+  split,
+  { intros h x hx hx',
+    rw ←mem_to_finset at hx hx',
+    exact h (finset.mem_inter_of_mem hx hx') },
+  { intros h x hx,
+    rw [finset.inf_eq_inter, finset.mem_inter, mem_to_finset, mem_to_finset] at hx,
+    exact h hx.left hx.right }
+end
```

Comment:
Do we have the lemma that `multiset.disjoint l l' ↔ l.disjoint l'`? If we did, we could prove this transitively from `disjoint_to_finset`